libs/cri-storing-service/src/main/java/uk/gov/di/ipv/core/library/cristoringservice/CriStoringService.java

```diff
@@ -120,7 +127,9 @@ public void storeVcs(
                     List.of(vc.serialize()), govukSigninJourneyId, ipAddress);
 
             verifiableCredentialService.persistUserCredentials(vc, criId, userId);
+            vcReceivedThisSession.add(vc.serialize());
```

Comment:
Originally we were only storing the VC ID, which I kinda liked for a couple of reasons:
- We're storing less data (a full VC is much bigger)
- Once we move to encrypted VC storage, we won't have any VCs in IPV Core controlled stores

If it's much easier to do this way I think it's not the end of the world.

Reply:
Ah yeah, this is the thing I mentioned in the PR comment. Turns out that not all CRIs are returning a JTI claim in their VCs, even though they're supposed to. Another option could be to just store the signature from the VC and use that as a unique identfier, but it's a bit meh.

Storing the whole thing gives us the option to just use the values in the session rather than going back to the table. I've left the implementation here to go and fetch from the store, bit only because I have a uneasy feeling it's safer. What do you reckon?

Reply:
Sorry, I should have read the description! Do we know which CRIs don't issue a JTI, and why not (can we report it as a bug?)

In any case, they probably won't fix it soon enough for us here :( - maybe we can add it as a tech debt change later on. Given that, I agree that re-fetching from the store feels better, although it shouldn't actually make any difference.

Reply:
I think it's a known about thing - it's mentioned in this RFC: https://github.com/govuk-one-login/architecture/blob/a6314577bcf8e49e6ff07b147b7ae51f749738e5/rfc/0041-CI-mitigation-VC.md?plain=1#L39
 
That RFC is 8 months old. Realisticly it's never getting fixed...

Reply:
Have created a tech-debt ticket to fix it up when we can: https://govukverify.atlassian.net/browse/PYIC-4375